### PR TITLE
Rebranding of AMP Ads & A4A -> AMPHTML ads

### PR DESCRIPTION
**Pull request by bpaduch**
Change references of "AMP Ads", "A4A" -> "AMPHTML Ads"
Fixes https://github.com/ampproject/docs/issues/804

cc: @gbender03

### CODE REVIEW

**Comment by bpaduch** [comment]
@jasti - ping

File: extensions/amp-a4a/amp-a4a-format.md [hunk 1]
```diff
@@ -15,24 +15,24 @@ limitations under the License.
 -->
```
Comment (jasti) [inline extensions/amp-a4a/amp-a4a-format.md]: I'm wondering if we should rename the directory and this file name?

File: extensions/amp-a4a/amp-a4a-internal.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ limitations under the License.
 <table>
```
Comment (jasti) [inline extensions/amp-a4a/amp-a4a-internal.md]: Same comment as above, should we rename the file and directory?

File: extensions/amp-a4a/amp-a4a-format.md (hunk 1 above)
Comment (bpaduch) [reply extensions/amp-a4a/amp-a4a-format.md]: Not sure if it'll matter; especially if the content will eventually be imported into ampproject.org.

File: extensions/amp-a4a/amp-a4a-internal.md (hunk 2 above)
Comment (bpaduch) [reply extensions/amp-a4a/amp-a4a-internal.md]: I'd say no. Breaks links and the extension itself is still a4a.

**Pull request merged**